style.css

```diff
@@ -1,414 +1 @@
-html,
```

Comment:
It's not have reason, I can delete it.


     On Monday, September 28, 2015 4:47 PM, Ruan Naude <notifications@github.com> wrote:
   

 In style.css:> @@ -1,414 +1 @@
> -html,
Why did you remove all styles from this file?
And also why didn't you just delete the entire file instead of leaving it empty?—
Reply to this email directly or view it on GitHub.